test/e2e/traceflow_test.go

```diff
@@ -184,6 +336,24 @@ func TestTraceflow(t *testing.T) {
 		}
 	}()
 
+	if intraNodeUDPDstIPTraceflow, err = data.crdClient.OpsV1alpha1().Traceflows().Create(context.TODO(), intraNodeUDPDstIPTraceflow, v1.CreateOptions{}); err != nil {
```

Comment:
can we have a for loop to create the different traceflow CRDs and register the deferred cleanup functions, instead of repeating the same code over and over again?

Reply:
Yes, I moved create() and delete() into a goroutine same as get().